app/assets/javascripts/rails_admin/custom/show_hide.js

```diff
@@ -15,10 +15,9 @@ $(document).on('rails_admin.dom_ready', function() {
 
   // Wrapper for all showing/hiding of sponsors
   function toggleSponsorField() {
+    var elements = $(".sponsorship_fields_toggle")
     var checkbox_id = $('.sponsorship_fields_toggler').find('input')[1].id
-    showOrHide(checkbox_id, 'standard_challenge_sponsor_field')
-    showOrHide(checkbox_id, 'standard_challenge_sponsor_logo_field')
-    showOrHide(checkbox_id, 'standard_challenge_sponsor_description_field')
+    elements.toArray().forEach(element => showOrHide(checkbox_id, element.id))
```

Comment:
You should be able to just call `elements.hide()` or `elements.show()` here, it shouldn't be necessary to iterate them.

If you refactor showOrHide slightly you could even just pass a class or id selector to the showOrHide method and just let it handle all of this for you

```
  function showOrHide(checkboxId, fieldToToggle){
    if(document.getElementById(checkboxId).checked){
      $(fieldToToggle).hide()
    }
    else{
      $(fieldToToggle).show()
    }
  }
```
Usage
```
showOrHide(checkbox_id, '.sponsorship_fields_toggle');
```